wagtail/contrib/forms/tests/test_models.py

```diff
@@ -183,6 +183,26 @@ def test_invalid_from_address(self):
         with self.assertRaises(ValidationError):
             make_form_page(from_address="not an email")
 
+    def test_string_representation_form_submission(self):
+        """
+        Ensure that a form submission can be logged / printed without error.
+        Broke when converting field to JSON - see #8927
+        """
+
+        self.client.post(
+            "/contact-us/",
+            {
+                "your_email": "bob@example.com",
+                "your_message": "hello world",
+                "your_choices": {},
+            },
+        )
+
+        self.assertGreaterEqual(FormSubmission.objects.count(), 1)
+
+        submission = FormSubmission.objects.first()
+        self.assertIn("hello world", submission.__str__())
```

Comment:
Suggestion: Call `str` rather than the magic method directly. It makes no difference, but it's clearer what you're trying to achieve.
```suggestion
        self.assertIn("hello world", str(submission))
```

Reply:
Good call thanks.